### ISSUE TITLE
when my image path is a mounted volume, the web get request fails..

### ISSUE THREAD

**Issue by sdetweil** [AdamMoses-GitHub/MMM-ImageSlideshow#7]
my imagepath = /mnt/buildserver/media/photos/savedimages

i can ls that from a MM terminal window

**Comment by 1dollarbob** [comment]
anyone find a solution to this?  I'm running into this also

**Comment by sdetweil** [comment]
only work around I have found is to mount the remote source as a volume and then make a link to a directory/folder IN the module directory

**Comment by 1dollarbob** [comment]
yeah, looks like only the module directory is available, I'll move my mount to modules
